typescript/cli/src/config/agent.ts

```diff
@@ -42,34 +107,57 @@ export async function createAgentConfig({
         const deployedBlock = await mailbox.deployedBlock();
         return deployedBlock.toNumber();
       } catch (err) {
-        logRed(
-          `Failed to get deployed block to set an index for ${chain}, this is potentially an issue with rpc provider or a misconfiguration`,
+        errorRed(
+          `❌ Failed to get deployed block to set an index for ${chain}, this is potentially an issue with rpc provider or a misconfiguration`,
         );
         process.exit(1);
       }
     }),
   );
+}
 
-  // @TODO: consider adding additional config used to pass in gas prices for Cosmos chains
-  const agentConfig = buildAgentConfig(
-    chains,
-    multiProvider,
-    addresses as ChainMap<HyperlaneDeploymentArtifacts>,
-    startBlocks,
-  );
+async function handleMissingInterchainGasPaymaster(
+  chainAddresses: ChainMap<ChainAddresses>,
+) {
+  for (const [chain, addressesRecord] of Object.entries(chainAddresses)) {
+    if (!addressesRecord.interchainGasPaymaster) {
+      warnYellow(`interchainGasPaymaster address is missing for ${chain}`);
+      const zeroIGPAddress = await confirm({
```

Comment:
I think we may need to skip the interaction if `--yes` is used - iirc hybrid chains won't have an IGP and we'll want to still be able to eventually generate agent configs in CI even if that's the case. We can just default to using the zero address in this case

wonder if a helper function that wraps `confirm` where you can pass in whether or not to auto-confirm would be nice - it could print the question but then another log right after saying that it's auto-confirming